Automattic/wp-calypso
# Help Center: Move directly to a package

## Summary

1. This PR adds Directly to the support options. We have moved the directly code into the help-center package.
2. Moves Spinner from calypso/components to packages/components.
3. Uses the new package in the help center.

## Testing
1. Apply D81418-code to get Directly eligibility for every user.
2. Pull the branch and run `cd apps/editing-toolkit/ && yarn dev --sync`
3. Go check the HelpCenter contact form and it should automatically go to Directly.

### Regression testing - important
1. Using a free user (user without any purchases) visit http://calypso.localhost:3000/help/contact.
2. Type something and send the question.
4. Directly popup should appear.

Related to #63655
